CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # ?? (??) - ??
 
+# 3.10.1 (2021-09-TODO) - Paris
```

Comment:
TODO

Reply:
I completely forgot about it indeed, it should be good now I hope.